drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,69 +1,114 @@
-function DragAndDropBlock(runtime, element) {
-
+function DragAndDropBlock(runtime, element, configuration) {
+    "use strict";
+    // Ensure "undefined" has not been redefined (though this unlikely and often impossible).
```

Comment:
@bradenmacdonald Do the LMS/Studio/Workbench runtimes contain code that redefines `undefined`? Why would one want to do that in the first place? (Just asking because I've never come across code that does this. I'm fine with conditionally resetting `undefined` here.)